src/translations/translations.d.ts

```diff
@@ -0,0 +1,392 @@
+interface Languages {
+  [key: string]: {
+    label: string;
+    lang: string;
+    translation: Translation;
+  };
+}
+
+/**
+ * Defines the structure of translations and integrates it with the react-i18n module.
+ * When useTranslate is called, it verifies if the provided string exists.
+ * If the string does not exist, a compile error is returned.
+ */
+interface Translation {
+  acsSlow: {
+    body: string;
+    title: string;
+  };
+  app: {
+    title: string;
+  };
+  ariaLabels: {
+    appLanguage: string;
+    close: string;
+    editCard: string;
+    editDonation: string;
+    editPsp: string;
+    informationDialog: string;
+    informationLink: string;
+    languageMenu: string;
+    loading: string;
+  };
+  cancelledPage: {
+    body: string;
+    button: string;
+  };
+  cartDetail: {
+    amount: string;
+    companyName: string;
+    description: string;
+    fiscalCode: string;
+    noticeNumber: string;
+  };
+  clipboard: {
+    copied: string;
+    copy: string;
+    edit: string;
+  };
+  DOMAIN_UNKNOWN: {
+    title: string;
+    buttons: {
+      close: string;
+      help: string;
+    };
+  };
+  DONATIONLIST_ERROR: string;
+  donationPage: {
+    click: string;
+    code: string;
+    description: string;
+    dismissCTA: string;
+    dismissDescription: string;
+    dismissTitle: string;
+    entity: string;
+    ioDescription: string;
+    modalBody2: string;
+    modalTitle: string;
+    openSection: string;
+    payNotice: string;
+    portfolio: string;
+    submitCard: string;
+    submitIO: string;
+    title: string;
+    volunteer: string;
+  };
+  errorButton: {
+    close: string;
+    help: string;
+    retry: string;
+  };
+  ErrorCodeDescription: string;
+  GENERIC_ERROR: {
+    body: string;
+    title: string;
+  };
+  general: {
+    and: string;
+  };
+  indexPage: {
+    description: string;
+    title: string;
+  };
+  inputCardPage: {
+    formErrors: {
+      cid: string;
+      cvv: string;
+      expirationDate: string;
+      name: string;
+      number: string;
+      required: string;
+    };
+    formFields: {
+      cid: string;
+      cvv: string;
+      expirationDate: string;
+      name: string;
+      number: string;
+    };
+    helpLink: string;
+    modal: {
+      description: string;
+      descriptionAE: string;
+      title: string;
+    };
+    title: string;
+  };
+  INVALID_CARD: {
+    body: string;
+    title: string;
+  };
+  INVALID_DECODE: {
+    body: string;
+    title: string;
+  };
+  INVALID_QRCODE: {
+    body: string;
+    title: string;
+  };
+  koPage: {
+    body: string;
+    button: string;
+    helpButton: string;
+    title: string;
+  };
+  mainPage: {
+    footer: {
+      accessibility: string;
```

Comment:
code review pipeline (build) fails, the translations are incomplete, for example for IT:
```
  "footer": {
      "accessibility": "Accessibilità",
      "help": "Aiuto",
      "pagoPA": "Visita il sito pagoPA"
    },
```
so `privacy` and `terms` are missing.
or is the interface/model wrong?
cc @a-mannia @thisisjp